lib/galaxy/util/__init__.py

```diff
@@ -570,7 +570,7 @@ def mask_password_from_url( url ):
 
 
 def ready_name_for_url( raw_name ):
-    """ General method to convert a string (i.e. object name) to a URL-ready
+    u""" General method to convert a string (i.e. object name) to a URL-ready
```

Comment:
This is [D302](http://www.pydocstyle.org/en/latest/error_codes.html): Use u”“” for Unicode docstrings. So, only for the ones containing Unicode, and yes, the linter should give an error.